app/templates/logos.hbs

```diff
@@ -18,15 +18,14 @@
   <p><a href="/images/brand/logos-ember.zip">Download logos (ZIP)</a></p>
 </section>
 
-
 <section>
   <h3>Primary Logo</h3>
 
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Ember-Dark.png"></li>
-    <li><img src="/images/brand/ember_Ember-Light.png"></li>
-    <li><img src="/images/brand/ember_Ember-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Ember-1c-Dark.png"></li>
+    <li><img alt="Ember" src="/images/brand/ember_Ember-Dark.png"></li>
+    <li><img alt="Ember" src="/images/brand/ember_Ember-Light.png"></li>
+    <li><img alt="Ember" src="/images/brand/ember_Ember-1c-Light.png"></li>
+    <li><img alt="Ember" src="/images/brand/ember_Ember-1c-Dark.png"></li>
```

Comment:
Each of these alt texts should be different and descriptive of the difference in the logos

Reply:
Can you please help me with the text that needs to be added to the alt attributes? Or can I just go ahead and just add them based on the image name?